recensio/theme/skins/recensio_theme/secondarynavportlets.pt

```diff
@@ -56,12 +56,12 @@
            i18n:translate="label_secondary_nav_journals">Zeitschriften</a>
       </li>
       <li class="last"
-         tal:define="is_current python:'search' in context.REQUEST.getURL()"
+         tal:define="is_current python:'latest-review-items' in context.REQUEST.getURL()"
          tal:attributes="id python:is_current and 'lastcurrent' or ''"
          >
         <a
-           tal:attributes="class python:is_current and 'active' or ''; href python:context.rezensionen.zeitschriften.absolute_url() + '/filtered-search?portal_type:list=Review+Journal&amp;portal_type:list=Review+Monograph&amp;portal_type:list=Review+Article+Journal&amp;portal_type:list=Review+Article+Collection&amp;portal_type:list=Review+Exhibition&amp;sort_order=reverse&amp;sort_on=effective'"
-           href="filtered-search?portal_type:list=Review+Journal&amp;portal_type:list=Review+Monograph&amp;portal_type:list=Review+Article+Journal&amp;portal_type:list=Review+Article+Collection&amp;portal_type:list=Review+Exhibition"
+           tal:attributes="class python:is_current and 'active' or ''; href python:context.rezensionen.zeitschriften.absolute_url() + '/latest-review-items'"
```

Comment:
@thet, heads up, I've changed the `href` here but it has been requested that this link should be “pulled up” into the main navigation. We're probably going to get a conflict here but it should be easily resolved by deleting this markup and using `latest-review-items` in the main navigation instead.